submissions/famuyiwaOlawumi/HTML/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta charset="UTF-8">
+        <title>xyzmart</title>
+</head>
+
```

Comment:
Remove the extra one or 2 lines spaces in this file.

Reply:
Ok